### ISSUE TITLE
Should " gem.add_development_dependency 'active_support' "  be  " gem.add_development_dependency 'activesupport' " ? 

### ISSUE THREAD

**Issue by ayonliu** [bobthecow/genghis#178]
file : genghisapp.gemspec , line: 21
Should " gem.add_development_dependency 'active_support' "  be  " gem.add_development_dependency 'activesupport' " ? 

 Wher i run "bundle install", i got this:
Fetching gem metadata from https://ruby.taobao.org/........
Fetching additional metadata from https://ruby.taobao.org/..
Resolving dependencies...
Could not find gem 'active_support (>= 0) ruby' in the gems available on this machine.

**Comment by bobthecow** [comment]
Yes. It's fixed in the `develop` branch, but that one's still a bit away from stable release.

**Comment by ayonliu** [comment]
Ok, thanks for the answer.
